progressbar/widgets.py

```diff
@@ -338,8 +339,8 @@ def __call__(self, progress, data):
             value = None
             elapsed = 0
         else:
-            value = values[-1] - values[0]
```

Comment:
This would effectively make the adaptive ETA use only 1 sample instead of the configured amount of samples. Can you explain what you wanted to achieve with this modification?